index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup Mobile Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <article>
+        <header>   
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
+            <p class="logo">My Logo</p>  
```

Comment:
-You did well using a `header` tag for this part of your page. To make your page more semantically correct, I suggest you wrap your logo and menu-icon in a `nav` tag 🙏

Reply:
Thank you for the feedback